pkg/orchestration/wiring/datadog/types.go

```diff
@@ -0,0 +1,57 @@
+package datadog
+
+import (
+	"github.com/atlassian/voyager"
+)
+
+type AlarmSpecType string
+type AlarmType string
+
+const (
+	Metric AlarmSpecType = "metric alert"
+)
+const (
+	CPU    AlarmType = "cpu"
+	Memory AlarmType = "memory"
+)
+
+type Alarm struct {
+	Name    string      `json:"name"`
+	Type    string      `json:"type"`
+	Query   string      `json:"query"`
+	Option  AlarmOption `json:"options"`
+	Message string      `json:"message"`
+}
+
+type ServiceInstanceSpec struct {
+	ServiceName voyager.ServiceName `json:"serviceName"`
+	Attributes  Alarm               `json:"attributes"`
+	Environment voyager.EnvType     `json:"environment"`
+	Region      voyager.Region      `json:"region"`
+}
+type AlarmOption struct {
+	Timeout           int             `json:"timeout_h,omitempty"`
+	locked            bool            `json:"locked,omitempty"`
```

Comment:
Did you want this as private?

Reply:
It would be weird to have a private serialized field, I think it's a typo